### PR TITLE
feat: add story section to project pages

### PR DESCRIPTION
**Pull request by blushi**
## Description

Closes: regen-network/regen-registry#1587

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [x] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### How to test

Please only look at the story section, those test-story projects have been created only for this purpose:
https://deploy-preview-1869--regen-registry.netlify.app/project/test-story-video
https://deploy-preview-1869--regen-registry.netlify.app/project/test-story-image
https://deploy-preview-1869--regen-registry.netlify.app/project/test-story-no-text
https://deploy-preview-1869--regen-registry.netlify.app/project/test-story-no-media

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by blushi** [comment]
@erikalogie please see testing instructions in the description

**Comment by flagrede** [comment]
@blushi @erikalogie  Should we use the one-column layout below md breakpoint? Looks a bit awkward between sm and sm.
image

File: web-registry/src/components/organisms/ProjectStorySection/ProjectStorySection.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+          overflow: { md: 'visible' },
+          pb: { xs: 19.75, sm: 25 },
+          ...(storyTitle &&
+            story &&
+            storyMedia && { pr: { tablet: 2.5, lg: 0 } }),
+        },
+      }}
+    >
+      <Grid container>
+        {storyTitle && story && (
```
Comment (flagrede) [inline web-registry/src/components/organisms/ProjectStorySection/ProjectStorySection.tsx:33]: Optional suggestion, having 2 child components here for readability:
`<StoryTitle />` and `<StoryMedia />`

**Review by flagrede — APPROVED** [review]
Nice job, the styling part was not easy for this one.

**Comment by erikalogie** [comment]
> @blushi @erikalogie Should we use the one-column layout below md breakpoint? Looks a bit awkward between sm and sm. image

Yes I agree that we should use the one column layout at this size

**Comment by blushi** [comment]
@erikalogie ready for another look

**Comment by erikalogie** [comment]
@blushi looks good!

**Review by wgwz — APPROVED** [review]
utACK

**Pull request merged**